data-type-conversion/data-type-conversion-task2.js

```diff
@@ -0,0 +1,5 @@
+var x = 1;
+var y = 2;
+var res2 = Boolean(x) + '' + y ; // Допишіть код, необхідно використовувати змінні x та y
```

Comment:
You used tuple quotes in previous task.
```suggestion
var res2 = Boolean(x) + "" + y ; // Допишіть код, необхідно використовувати змінні x та y
```